amida-tech/orange-api
# Task/remove auth endpoints

# What does this do
This removes all the old auth endpoints in the orange code and the associated tests. Since we are now supporting generic JWT based authentication from an external identity provider/jwt issuer, this can all be classified as dead code.

# JIRA tickets this addresses
https://jira.amida-tech.com/browse/ORANGE-467

# How to test it
Use the app and verify nothing is broken. Note: the extAuth methods in the app still refer to auth endpoints in orange-api, these will need to be changed to point to the auth microservice external auth endpoints.
